spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+RSpec.feature 'Users Index View', type: :feature do
+  scenario 'User views users index page' do
+    user1 = User.create(name: 'Alice', photo: 'https://example.com/alice.jpg')
+    user1.posts.create(title: 'Post 1', text: 'First post by Alice')
+    user1.posts.create(title: 'Post 2', text: 'Second post by Alice')
+    user2 = User.create(name: 'Bob', photo: 'https://example.com/bob.jpg')
+    user2.posts.create(title: 'Post 3', text: 'First post by Bob')
+    visit users_path
+    expect(page).to have_selector('.container')
+    expect(page).to have_selector('.user_info', count: 4)
```

Comment:
- [ ] A user index view spec is failing. Let's ensure all spec scenarios are passing!

**Failing spec:**
![image](https://github.com/Obimbo07/tdn/assets/12745474/49a0345a-7267-4025-aec6-eb4045a3a744)